### DISCUSSION TITLE
Are the columns in helm-mini adjustable?

### DISCUSSION THREAD

**Discussion by pedz** [emacs-helm/helm#2567] [Q&A] [answered]
I have a lot of buffers produced by a package called Yari.  They are essentially output from Ruby's `ri` command.  A typical name of the buffer is `*yari ActionDispatch::Flash::FlashHash#each*`.  When I use `helm-mini` I can only see the first part of the buffer name.  (See attached).

Is there a way to adjust the columns?

<img width="710" alt="Screenshot 2022-11-01 at 17 21 52" src="https://user-images.githubusercontent.com/76111/199353838-f64853bc-4791-494b-acb2-7fa57925dd87.png">

**Comment by thierryvolpiatto** [comment, marked as answer]
Perry Smith ***@***.***> writes:

> I have a lot of buffers produced by a package called Yari. They are essentially output from Ruby's ri command. A typical name of the buffer is *yari ActionDispatch::Flash::FlashHash#each*.
> When I use helm-mini I can only see the first part of the buffer name. (See attached).
>
> Is there a way to adjust the columns?

Yes, adjust `helm-buffer-max-length`.
You can also toggle details with C-] to see the whole buffer name only.

> Screenshot 2022-11-01 at 17 21 52                                      
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.*Message ID: ***@***.***>


-- 
Thierry
